scripts/integration-tests/utils/bump-babel-dependencies.js

```diff
@@ -5,23 +5,23 @@ const packageJSONPath = path.resolve(cwd, "./package.json");
 const content = JSON.parse(fs.readFileSync(packageJSONPath));
 
 let bumped = false;
-function bumpBabelDependency(dependencies) {
+function bumpBabelDependency(dependencies, version) {
   for (const dep of Object.keys(dependencies)) {
     if (dep.startsWith("@babel/") && !dependencies[dep].includes(":")) {
-      dependencies[dep] = "latest";
+      dependencies[dep] = version;
       bumped = true;
     }
   }
 }
 
 if ("peerDependencies" in content) {
-  bumpBabelDependency(content.peerDependencies);
+  bumpBabelDependency(content.peerDependencies, "*");
```

Comment:
This is because `yarn` was throwing for `latest` in the `peerDependencies`

Reply:
Or we can simply remove `peerDependencies` 😈.

Reply:
It will cause problems if one of the packages we test starts using PnP :stuck_out_tongue: